iib/workers/config.py

```diff
@@ -42,6 +42,11 @@ class Config(object):
     # Don't allow the worker to fetch more messages than it can handle at a time. This is so that
     # other tasks aren't starved. This will only be useful once more workers are enabled.
     worker_prefetch_multiplier = 1
+    grpc_max_tries = 5
```

Comment:
Could you please prefix these with `iib_` and move it up alphabetically. Also, a comment explaining the use of these variables would be great. 

Also, could you please update the README with these variables?

Reply:
Done.